SCClassLibrary/Common/Control/Server.sc

```diff
@@ -351,13 +353,13 @@ Server {
 
 	}
 
+	numClients { ^numClients ?? { options.maxLogins } }
```

Comment:
in such simple cases it is more efficient to write `numClients ? options.maxLogins`

Reply:
done